js/shop.js

```diff
@@ -104,131 +28,72 @@ if (authorized){
-                  '</p></div><div class="shop-btn-right">' +
-                  '<a class="btn btn-small btn-primary" href="' + add_shopper_url + '">Add Course</a></div>';
-
-              var shopping_is_active_message = '<h1>Your current class list may include Shoppers. ' + tooltip_link +
-                  '</h1><p>All Harvard ID holders can view this course site during shopping period. Students ' +
-                  'can choose to add themselves as Shoppers to participate in discussions, upload assignments, watch ' +
-                  'lecture videos, and receive notifications for this course before they are officially enrolled. '+
-                  'Student contributions will be visible to other students who are also shopping this course. At the '+
-                  'end of shopping period, Shoppers who have not officially enrolled as Students or Guests in the '+
-                  'course through the Registrar’s office will be removed from the class list.</p>';
-
-              var manage_shopping_li_item = jQuery('<li/>', {
-                id: 'manage-shopping',
-                class: 'menu-item'
-              });
-
-              var manage_shopping_link = jQuery('<a/>', {
-                id: 'manage-shopping-page-link',
-                class: 'menu-item-no-drop',
-                href: manage_shopping_page_url,
-                text: "Courses I'm Shopping"
-              });
-
-              /*
-               build the Manage Shopping menu item
-               */
-              if (user_enrolled) {
-                manage_shopping_li_item.append(manage_shopping_link);
-
-                /*
-                 for each role format the appropriate banner
-                 */
-                if (is_shopper) {
-                  $("ul#menu").append(manage_shopping_li_item);
-                  shopping_banner.append(shopper_message_text);
-                }
-                else if (is_teacher) {
-                  shopping_banner.append(shopping_is_active_message);
-                }
-                /*
-                 display the banner formatted above
-                 */
-                if (is_shopper || is_teacher) {
-                  $('#breadcrumbs').after(shopping_banner);
-                }
-              }else{
-                /*
-                 If logged in user is not enrolled, then display generic shopping message to authorized user
-                 */
+            if (user_enrolled) {
+              manage_shopping_li_item.append(manage_shopping_link);
+              if (is_shopper) {
                 $("ul#menu").append(manage_shopping_li_item);
-                shopping_banner.append(viewer_message_text);
+                shopping_banner.append(shopping_get_shopper_banner_text(remove_shopper_url));
+              }
+              else if (is_teacher) {
+                shopping_banner.append(shopping_get_is_active_banner_text());
```

Comment:
Minor: comment in other PR to keep it uniform with the other method names and call this shopping_get_teacher_banner_text as it seems to be shown only to Teachers.